src/main/resources/swagger/api-docs.yaml

```diff
@@ -1050,6 +1050,11 @@ definitions:
       machineConfig:
         description: The machine configurations for the master and worker nodes
         $ref: '#/definitions/MachineConfig'
+      properties:
+        description: Example `{"spark:spark.executor.memory": "10g"}`. See https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/cluster-properties for allowed property settings
```

Comment:
is the example properly escaped? yamllint didn't seem to like it. i think the colon in the link will be a problem too